src/fides/api/schemas/privacy_preference.py

```diff
@@ -140,6 +150,14 @@ def tcf_duplicates_detected(preference_list: List) -> bool:
                 raise ValueError(
                     f"Duplicate preferences saved against TCF component: '{field_name}'"
                 )
+
+        if values.get("tc_string"):
```

Comment:
so i'm assuming we're deliberately deciding to pack all of this into one endpoint/request model, i.e. `PATCH /privacy-preferences` is just a single endpoint and we have this logic to make sure there aren't overlaps; as opposed to having a distinct enpdoint (or query param) and model to save preferences via TC string vs. "traditional" fields.

is the idea that we do want to support some of the other traditional preference fields alongside the TC string? just playing devil's advocate a bit and wondering whether we considered a slightly different design, where we split out the TC string saving method a bit more and we don't totally overload the "traditional" preference save.

Reply:
OK merging for now, but we can separate out later if needed!